### ISSUE TITLE
Cisco UCS Plugin

### ISSUE THREAD

**Issue by thebsdbox** [docker/infrakit#685]
To track the Cisco UCS Plugin. 

- [x] Examining existing SDKs
- [ ] PoC of creating Server instances (In progress)
- [ ] Code aligned with providers location

**Comment by GuillaumeMorini** [comment]
Hi, do you need help on this one ?
I am also interested in having this, and could help if needed

**Comment by thebsdbox** [comment]
Hello @GuillaumeMorini, Always happy to have assistance around the plugins. I'd be happy for any help that you can provide, i'm pretty behind schedule on building out this plugin at the moment.

**Comment by micdoher** [comment]
Hi Daniel, will you be at Dockercon next week? The reason I ask is I work with Guillaume and the team helping to align all the container and DevOps tools for our Data Center teams. I would really like to discuss what the infrakit could bering to UCS in alignment with the other efforts within Cisco currently going on. I will be on the Cisco booth.

**Comment by thebsdbox** [comment]
Hello @micdoher I will be at DockerCon next week, so if you have some time it would be good to align and discuss the plan I had for InfraKit and a Cisco UCS plugin.

**Comment by micdoher** [comment]
Yes please! I will be on the Cisco booth for much of the event if you could pass by please. Alternatively I can come and find you?

**Comment by thebsdbox** [comment]
cc/ @GuillaumeMorini & @micdoher 

Here is the initial Cisco UCS InfraKit plugin -> #743

**Comment by GuillaumeMorini** [comment]
Thanks Daniel
Any guide to start ?
Typically how to build it and test it ?

Guillaume

On 2 Nov 2017, at 17:23, Daniel Finneran <notifications@github.com<mailto:notifications@github.com>> wrote:


cc/ @GuillaumeMorini<https://github.com/guillaumemorini> & @micdoher<https://github.com/micdoher>

Here is the initial Cisco UCS InfraKit plugin -> #743<https://github.com/docker/infrakit/pull/743>

**Comment by thebsdbox** [comment]
@GuillaumeMorini  - I will write up some documentation today !
